mm2src/mm2_io/src/fs.rs

```diff
@@ -233,6 +264,15 @@ where
     Ok(result)
 }
 
+/// Read the `dir_path` entries trying to deserialize each as the `T` type from JSON files.
+/// Please note that files that couldn't be deserialized are skipped.
+pub async fn read_dir_json<T>(dir_path: &Path) -> FsJsonResult<Vec<T>>
+where
+    T: DeserializeOwned,
+{
+    read_files_with_extension(dir_path, "json").await
+}
```

Comment:
nit: Do we really need this function?

Reply:
`read_dir_json` is used for orders and swaps and I wanted to keep it the same way instead of passing `"json"` everywhere it's used.

Reply:
Let's add `inline(always)` attribute then.

Reply:
Added `inline(always)` here https://github.com/KomodoPlatform/komodo-defi-framework/pull/2202/commits/e1dbb67da0d66aff783ab63a8449896b8d8f9728